python_modules/dagit/setup.py

```diff
@@ -44,15 +44,18 @@ def get_version():
     include_package_data=True,
     install_requires=[
         # cli
-        "click>=7.0,<9.0",
         f"dagster{pin}",
+        f"dagster-webserver{pin}",
         f"dagster-graphql{pin}",
-        "starlette",
-        "uvicorn[standard]",
     ],
     extras_require={
         "notebook": ["nbconvert"],  # notebooks support
         "test": ["starlette[full]"],  # TestClient deps in full
     },
```

Comment:
should this only depend on  `dagster-webserver{pin}` ? 

and 
below just point at `dagster-webserver[full]` / `dagster-webserver[test]`

Reply:
Yes-- except `dagster-webserver[notebook]` and `dagster-webserver[test]` (there is no `dagster-webserver[full]`.